master/job.go

```diff
@@ -10,8 +10,14 @@ import (
 
 type job struct {
 	lib.Job
-	id    int
-	tasks []*lib.Task
+	id           int
+	tasks        []*lib.Task
+	runningTasks int
```

Comment:
you could iterate over all the tasks and find how many of them are running right?